src/main/java/com/cburch/logisim/fpga/prefs/SoftwaresOptions.java

```diff
@@ -214,12 +222,36 @@ public SoftwaresOptions(PreferencesFrame window) {
     c.weightx = 1.0;
     c.fill = GridBagConstraints.HORIZONTAL;
     add(vivadoPathField, c);
-
     c.gridx = 2;
     c.gridy = gridY++;
     c.fill = GridBagConstraints.NONE;
     add(vivadoPathButton, c);
 
+    c.gridx = 0;
+    c.gridy = gridY++;
+    c.gridwidth = 4;
+    c.weightx = 1.0;
+    c.fill = GridBagConstraints.HORIZONTAL;
+    add(new JSeparator(JSeparator.HORIZONTAL), c);
+
+    c.gridx = 0;
+    c.gridy = gridY++;
+    c.gridwidth = 4;
+    c.weightx = 1.0;
+    c.fill = GridBagConstraints.HORIZONTAL;
+    add(openfpgaPathLabel, c);
+
+    c.gridx = 0;
+    c.gridy = gridY;
+    c.gridwidth = 2;
+    c.weightx = 1.0;
+    c.fill = GridBagConstraints.HORIZONTAL;
+    add(openfpgaPathField, c);
```

Comment:
It is not clear to me why some `add` operations are followed by blank lines and others are not. You deleted a blank line above but added or left others. I can see that the blank lines block things for readability, but it should be done consistently.

Reply:
It was like this before, and I agree it is not very consistent, but I keep it as "beyond the scope of this PR".